src/System.Reflection.Metadata/src/System/Reflection/Metadata/PortablePdb/PortablePdbVersions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Reflection.Metadata
+{
+    internal static class PortablePdbVersions
+    {
+        /// <summary>
+        /// Version of Portable PDB format emitted by the writer by default. Metadata version string.
+        /// </summary>
+        internal const string DefaultMetadataVersion = "PDB v1.0";
+
+        /// <summary>
+        /// Version of Portable PDB format emitted by the writer by default.
+        /// </summary>
+        internal const ushort DefaultFormatVersion = 0x0100;
+
+        /// <summary>
+        /// Minimal supported version of Portable PDB format.
+        /// </summary>
+        internal const ushort MinFormatVersion = 0x0100;
+
+        /// <summary>
+        /// Minimal supported version of Embedded Portable PDB blob.
+        /// </summary>
+        internal const ushort MinEmbeddedVersion = 0x0100;
+
+        /// <summary>
+        /// Version of Embedded Portable PDB blob format emitted by the writer by default.
+        /// </summary>
+        internal const ushort DefaultEmbeddedVersion = 0x0100;
+
+        /// <summary>
+        /// Minimal version of the Embedded Portable PDB blob that the current reader can't interpret.
+        /// </summary>
+        internal const ushort MinUnsupportedEmbeddedVersion = 0x0200;
```

Comment:
I removed it. Was thinking about versioning the blob as major = breaking, minor = non-breaking, but then couldn't think of a non-breaking change that could possibly work, so abandoned the idea and instead assume any version change is breaking. Will delete this const